### PR TITLE
Weekly Meeting, 2018-05-24

### PR DESCRIPTION
**Pull request by graeme-a-stewart**

### CODE REVIEW

File: organization/_posts/2018-05-24-coordination.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    -   Well received. Discussion focused on licenses and R&D in
+        simulation.
+    -   EIC happy to give us an HSF talk in return on their activities.
+-   [IEEE eScience
+    conference](https://www.escience2018.com/), Exascale
+    Computing in High-Energy Physics track: abstract deadline extended
+    to June 18.
+-   Presentation by Benedikt at Belle II software week. Feedback about
+    where more collaboration is possible and desired. For licensing
+    just following what the community decides.
```
Comment (hegner) [inline organization/_posts/2018-05-24-coordination.md]: my bad. Forgot to put that in initially: Copyright holder unclear. CERN was brought up as potential community-wide code owner.

File: organization/_posts/2018-05-24-coordination.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+          more highlighting of the more active areas.
+    -   More variation in meeting themes (more topical):
+        -   DD4hep meeting confirmed for 6 June.
+        -   Do we restart this in the Software Tech Forum?
+            -   Torre: that never seemed to work too well - rethink this
+                activity.
+            -   Links to HSF were weaker than they should be.
+    -   Collaboration guide for projects
+        -   Draft by Benedikt. Iterating with Graeme.
+-   Are we missing the big picture for "software upgrades"?
```
Comment (hegner) [inline organization/_posts/2018-05-24-coordination.md]: I would emphasize a bit more that this was the big discussion this time. Don't have a good idea how though

File: organization/_posts/2018-05-24-coordination.md (hunk 1 above)
Comment (graeme-a-stewart) [reply organization/_posts/2018-05-24-coordination.md]: Should I add anything about KEK and DESY being other options?

Comment (hegner) [reply organization/_posts/2018-05-24-coordination.md]: Hmm... leave that out for now.

**Comment by graeme-a-stewart** [comment]
URL failures from http://hepsoftware.org being down.

**Pull request merged**